src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -106,6 +108,8 @@ public class ServerStore : IDisposable
 
         public Operations Operations { get; }
 
+        public ConcurrentDictionary<string, long > LastRaftIndex = new ConcurrentDictionary<string, long>();
```

Comment:
What happens to this after server restart? It would be empty, should we fill it?